### ISSUE TITLE
[Feature]: Put Trace First in HTML Report

### ISSUE THREAD

**Issue by Georgegriff** [microsoft/playwright#36141]
### 🚀 Feature Request

We're building some accessibility tests which can generate lots of screenshots when there is a violation, as such our reports can get quite long and the extremely valuable trace can get hidden at the bottom after lots of screenshots, could you perhaps consider re-ordering the trace in the html report so it comes `before` the screenshots instead of after?

This might make sense anyway because then it goes

- Test Steps
- Trace
- Screenshots
- Videos
- Attachments

instead of the current:

- Test Steps
- Screenshots
- Trace
- Videos
- Attachments

### Motivation

Playwright best feature (the trace) at the top of reports

**Comment by pavelfeldman** [comment]
We can probably put a shortcut to the trace high on top.

**Comment by deveshruttala-edu** [comment]
Mock comment: Hyperbrowser can help with this issue by providing AI-native browsing support.
